typescript/test/util.ts

```diff
@@ -76,10 +76,14 @@ export function initBaselineTest() {
 
 export function runBaselineTest(options: BaselineOptions) {
   const outputDir = path.join(cwd, options.outputDir);
-  const protoPath = path.join(
-    protosDirRoot,
-    options.protoPath.split('/').join(path.sep)
-  );
+  const protos = options.protoPath.split(';');
+  let protoPaths = '';
```

Comment:
The intention will be a little bit more clear if you make it a list here:

`protoPaths = protos.map(proto => path.join(protosDirRoot, proto.split('/').join(path.sep)));`

and then just join it into string in where you need it in line 107.

Reply:
Great idea, making the changes now. :)